manifests/config.pp

```diff
@@ -74,6 +75,19 @@
     }
   )
 
+  # The --no-logger flag that disables syslog output must either be present or
+  # not at all (there is no --logger) flag. This means a mesos::property can't
+  # be used.
```

Comment:
I don't really understand what this comment is trying to express. Does it mean that `logger` and `no-logger` are incompatible? That the presence of one implies the absence of the other?